packages/fxa-settings/src/components/PageSettings/index.tsx

```diff
@@ -13,18 +13,38 @@ import LinkedAccounts from '../LinkedAccounts';
 import * as Metrics from '../../lib/metrics';
 import { useAccount } from '../../models';
 import { DeleteAccountPath } from 'fxa-settings/src/constants';
-import { Localized } from '@fluent/react';
+import { Localized, useLocalization } from '@fluent/react';
 import DataCollection from '../DataCollection';
 
 export const PageSettings = (_: RouteComponentProps) => {
   const { uid } = useAccount();
+  const { l10n } = useLocalization();
 
   Metrics.setProperties({
     lang: document.querySelector('html')?.getAttribute('lang'),
     uid,
   });
   Metrics.usePageViewEvent(Metrics.settingsViewName);
 
+  // Temp test to see how often we actually display fallback text
+  useEffect(() => {
+    Metrics.logViewEvent(Metrics.settingsViewName, 'test.fallback.start');
+    const ftlId = 'app-footer-mozilla-logo-label';
```

Comment:
I was not expecting a hard coded ID here. From the ticket it sounded like we really wanted to know the percentage of l10n ids that weren't found, thereby resulting in fallback text being shown. 

Doing this it seems like you'll get a sample on one specific ID, which maybe is enough, but I could also see this resulting in a skewed statistic. 

Anyways, just double checking that this approach is really inline with what the issue is asking for.

Reply:
I'm more so wondering if, or how often, Fluent fails to pull in a string from an ID we have, which would result in fallback text being shown. By "percentage of l10n ids that weren't found" I'm not sure if you mean essentially creating a list of all Settings IDs used and then looping through each one and using `l10n.getString` to check the ID and grab the values then send stats up on that, or if you mean to check FTL IDs we reference in TSX and compare them to the IDs found in the FTL file to catch IDs we should have in FTL but don't, but I think either one of those are pretty heavy.

I get what you're saying though, it's not a perfect test. We can circle back here if this produces weird stats.